com/company/Encryptor/Encryptor.java

```diff
@@ -0,0 +1,43 @@
+package com.company.Encryptor;
+
+public class Encryptor {
+
+    public String encryptionPassword(Users users) {
```

Comment:
refactor to encryptUserPassword(String userPassword),
in the method body you don't need to have User class,
you are only working with user's password.